cpp/src/io/text/multibyte_split.cu

```diff
@@ -418,12 +418,15 @@ std::unique_ptr<cudf::column> multibyte_split(cudf::io::text::data_chunk_source
   auto reader = source.create_reader();
   reader->skip_bytes(relevant_offset_first);
 
-  auto relevant_bytes = reader->get_next_chunk(string_chars_size, stream);
-
-  thrust::copy(rmm::exec_policy(stream),
-               relevant_bytes->data(),  //
-               relevant_bytes->data() + relevant_bytes->size(),
-               string_chars.begin());
+  for (int32_t i = 0; i < string_chars_size; i += ITEMS_PER_CHUNK) {
+    auto const read_size   = std::min<int64_t>(ITEMS_PER_CHUNK, string_chars_size - i);
+    auto const chunk_bytes = reader->get_next_chunk(read_size, stream);
+
+    thrust::copy(rmm::exec_policy(stream),
+                 chunk_bytes->data(),
+                 chunk_bytes->data() + chunk_bytes->size(),
+                 string_chars.begin() + i);
```

Comment:
Is this a sync copy? If yes, is there any chance it can be improved by async copy like `cudaMemcpyAsync`?

Reply:
maybe. we can test that in another iteration. it may (slightly) benefit from multiple streams as well.